Project.toml

```
@@ -5,17 +5,24 @@ version = "0.8.0"
 
 [deps]
 BitIntegers = "c3b6d118-76ef-56ca-8cc7-ebb389d030a1"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 
 [compat]
+Aqua = "0.8"
 BitIntegers = "0.2, 0.3"
+Compat = "3.47.0, 4.10.0"
+Dates = "1"
 FilePathsBase = "0.9.13"
+Printf = "1"
+Test = "1"
```

Since we require Julia 1.4 as the minimum and that predates versioned stdlibs (IIRC), I think we'll need this:
```suggestion
Dates = "0, 1"
FilePathsBase = "0.9.13"
Printf = "0, 1"
Test = "0, 1"
```

---

note that CI already passes on 1.4 😉

---

Right, it's for the registry

---

The registry CI does

---

But more specifically, where this matters is in the registry's dependency tracking file (Deps.toml I think it is): the stdlib versions need to be recorded such that they allow `<0.0.0` if support for earlier Julia versions is needed.